test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -921,17 +1117,64 @@ public static void generateClassFile(String name, byte[] bytes) {
 	}
 
 	public static Class<?> generateValueClass(String name, String[] fields) throws Throwable {
-		byte[] bytes = generateClass(name, fields, false, false);
```

Comment:
instead of duplicating the body, this can  call `generateValueClass(String name, String[] fields, String nestHost)` with a null nest host

Reply:
Addressed in commit fbba19114